hip-service-test/Link/Builder/TestBuilder.cs

```diff
@@ -0,0 +1,10 @@
+using Bogus;
+
+namespace hip_service_test.Link.Builder
+{
+    public class TestBuilder
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4795709)